data/env/Makefile.am

```diff
@@ -13,25 +13,12 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-SNAP_MOUNT_DIR := /snap
-ENVD := /etc/profile.d
-
 %.sh: %.sh.in
 	sed   < $< > $@ \
-		s:@SNAP_MOUNT_DIR@:${SNAP_MOUNT_DIR}:g
-
-GENERATED = snapd.sh
-
+		s:[@]SNAP_MOUNT_DIR[@]:${SNAP_MOUNT_DIR}:g
```

Comment:
Nitpick, it would be good if all sed invocations followed one style.

Reply:
Updated all `sed` calls to use the same construct